### PR TITLE
[Main Course] - Kaggle optiver

### PR DESCRIPTION
**Pull request by jhqian0909**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Post for kaggle optiver

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update

# How Has This Been Tested?

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: docs/Main_Course/Data_Analytics/2021-08-27-Kaggle-optiver.md [hunk 1]
```diff
@@ -0,0 +1,466 @@
...
+在金融范畴中，波动率有不同的衡量方式，如最简单的股票收益率的标准差，也有其他的，如高频中的已实现波动率，隐含波动率等。  
+赛题是一个回归问题，给出了10分钟的订单薄数据，需要我们预测未来10分钟的已实现波动率：  
+
+$$ \sigma=\sqrt{\sum_{t} r_{t-1, t}^{2}} $$
+
+其中，r是股票的log收益率，即股价取log做差分。评价指标是RMSPE(root mean square percentage error)：  
+
+$$ \mathrm{RMSPE}=\sqrt{\frac{1}{n} \sum_{i=1}^{n}\left(\left(y_{i}-\hat{y}_{i}\right) / y_{i}\right)^{2}} $$
+
+## 4 具体代码
```
Comment (jz263) [inline docs/Main_Course/Data_Analytics/2021-08-27-Kaggle-optiver.md:27]: 小标题序号好像有点问题

**Review by jz263 — COMMENTED** [review]
ad a comment

**Review by Realvincentyuan — APPROVED** [review]
👍

File: docs/Main_Course/Data_Analytics/2021-08-27-Kaggle-optiver.md (hunk 1 above)
Comment (jhqian0909) [reply docs/Main_Course/Data_Analytics/2021-08-27-Kaggle-optiver.md]: 已修改

**Pull request merged**